### ISSUE TITLE
Can Changelog link to the PR?

### ISSUE THREAD

**Issue by rebeccafae** [rubocop/rubocop-capybara#128]
In the changelog, can it link to the PR that made the change? It would make it a lot easier to find some of these to better see the impact it is having.

**Comment by pirj** [comment]
I support the idea for all our repos @rubocop/rubocop-rspec WDYT?

**Comment by bquorning** [comment]
I like the idea too. The main downside is that you don’t know the PR number when you write the changelog entry. It would be more efficient maybe to add the links right before releasing a new version?

**Comment by rebeccafae** [comment]
> I like the idea too. The main downside is that you don’t know the PR number when you write the changelog entry. It would be more efficient maybe to add the links right before releasing a new version?

You don't? Doesn't the line get added in the same PR as the change, like in this one? https://github.com/rubocop/rubocop-capybara/pull/99

**Comment by bquorning** [comment]
Yes, the changelog lines get added in each PR. What I meant was that the contributor don’t know the number of the PR until it’s opened, which is _after_ they push their code. So if we require the PR number to be in the changelog before we can merge a PR, each contributor _has_ to push to their branch twice: Once before opening the PR and (at least) once after opening the PR.

What I’m saying is that I would very much like to still be able to make one-push pull requests.

**Comment by r7kamura** [comment]
I also feel the desire to follow the PR links from the changelog.

Instead of manually writing changes in CHANGELOG.md, I am considering that it would be good to migrate to GitHub Releases and utilize the automatically generated release notes.

- https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes

This would reduce the burden on those creating PRs compared to before, and also solve other problems such as CHANGELOG.md frequently causing conflicts.
